recipes/xtrabackuprb.rb

```diff
@@ -1,10 +1,8 @@
 include_recipe 'git'
 include_recipe 'osl-postfix'
 include_recipe 'osl-mysql'
-include_recipe 'percona::package_repo'
 include_recipe 'yum-epel' if node['platform_version'].to_i == 6
-
-package 'percona-xtrabackup'
+include_recipe 'percona::backup'
```

Comment:
Before you include this recipe, you need to set ``node.default['percona']['skip_passwords'] = true`` before you include this recipe as it's causing an issue when converging the xtrabackuprb and backup suites.

Reply:
Somehow this didn't trigger for me when running the tests last week but I saw it now. Thanks!